map/src/i18n/langs/en.json

```
@@ -1,5 +1,6 @@
 {
   "title": "COVID-19 Mutual Aid Map",
+  "inFrameTitle": "View this map on {map}",
```

In most cases, there won't be the context of Reach4Help when embedded, so people won't necessarily understand that this link will allow them to find more information about the map, or add information.

I suggest we do this instead:

```suggestion
  "inFrameLink": "About this map / add information",
```